.vitepress/config.ts

```diff
@@ -665,6 +665,16 @@ export default defineConfigWithTheme<ThemeConfig>({
         text: 'Français',
         repo: 'https://github.com/vuejs-translations/docs-fr'
       },
+      {
+        link: 'https://ko.vuejs.org',
+        text: '한국어',
+        repo: 'https://github.com/vuejs-translations/docs-ko'
+      },
+      {
+        link: 'https://pt.vuejs.org',
+        text: 'Português',
+        repo: 'https://github.com/vuejs-translations/docs-pt'
+      },
```

Comment:
You missed updating the `src/translations/index.md` on this commit: https://github.com/vuejs/docs/commit/75ed0d45dfb48b8e7cd62e841dd709dc5a050da2